configure.ac

```diff
@@ -89,14 +89,14 @@ AM_CONDITIONAL(SYSTEMD, test -n "${with_systemdunitdir}")
 AC_ARG_WITH([dbuspolicydir],
         AS_HELP_STRING([--with-dbuspolicydir=DIR], [D-Bus policy directory]),
         [],
-        [with_dbuspolicydir=${datadir}/dbus-1/system.d])
+        [with_dbuspolicydir="$($PKG_CONFIG --variable=datadir dbus-1)/dbus-1/system.d"])
```

Comment:
What happens here if dbus-1.pc is not available?

Reply:
Not tried it yet, but I guess this would break the installation since $(...) expands to empty string. Do you suggest to add ${datadir} as fallback for example?

Reply:
Maybe we should add a check before and fail when dbus-1 package is not available... This makes the problem more obvious to the user than using a silent fallback.

Reply:
Yes, failing if dbus-1 is missing makes sense.

Reply:
Added a check in first patch, rebase to latest master.

Reply:
The build now fails at exactly this place, so the new dependency needs to be added to the CI scripts and documentation.

Reply:
I guess I need help here... I added the devel package name to Dockerfile and Travis script, but I've no glue how this is pulled in so that this PR builds successfully. Sorry, I did not work with Docker before and I'm not sure that I understand the CI setup completely. I guess that the Docker image needs a rebuild before Travis pulls it here and can succesfully use it?

Reply:
It's not that easy, because the container is build independently from master branch by docker hub and there is no other clean way I would know than a 2-step process of mergingfor adding such a dependency.

The only other solution here would be to test this manually. @jluebbe what do you think?

This is the compromise we had to accept when moving to Docker-based testing, but it's worth the drawback as testing became much more stable since this.

Reply:
We've updated the Dockerfile, and rebuilt. So the checks succeed now.